packages/driver/src/cy/commands/navigation.ts

```diff
@@ -156,13 +110,13 @@ const aboutBlank = (cy, win) => {
   })
 }
 
-const navigationChanged = (Cypress, cy, state, source, arg) => {
+const navigationChanged = async (Cypress, cy, state, source, arg) => {
   // get the current url of our remote application
-  const url = cy.getRemoteLocation('href')
+  const url = await cy.getCrossOriginRemoteLocation('href')
```

Comment:
I wonder if we can make the types for this type of function clearer, as well as some of the ones in the `locationHelper`. Probs not necessary here, but wondering if its a good idea to help with first timers reading through the code to get a better idea of what is going on